doc/api/fs.md

```diff
@@ -3532,6 +3530,10 @@ to the completion callback.
 Using `fs.rmdir()` on a file (not a directory) results in an `ENOENT` error on
 Windows and an `ENOTDIR` error on POSIX.
 
+Setting `recursive` to `true` results in behavior similar to the rimraf package
```

Comment:
I think it would be OK to drop the reference to `rimraf` and just explain this function's behavior.

Reply:
My concern here is that `rimraf` implies a lot of behavior, perhaps we can work together and figure out the most important behavioral things to call out? I chose two things that stood out for me, from what @RyanZim and @CxRes mentioned.

Reply:
My personal point of view: I don't think I've ever directly used the `rimraf` module in my own code, so referencing it here would force me to go read about implied behavior of an npm module in order to understand how core works. That feels a little backward.

I don't feel strongly about it though. If you think it's important to note, then it's fine with me.

Reply:
@cjihrig I looked at the `rimraf` docs, they point out that `rimraf` has behavior similar to `rm -rf`. Why don't we use this language instead of calling out the module name?

Reply:
That sounds good to me 👍